api/sdk.api.md

```diff
@@ -49,18 +49,16 @@ export class Confidence implements EventSender, Trackable, FlagResolver {
     withContext(context: Context): Confidence;
 }
 
+// Warning: (ae-forgotten-export) The symbol "UrlOptions" needs to be exported by the entry point index.d.ts
+//
 // @public
-export interface ConfidenceOptions {
+export type ConfidenceOptions = {
     clientSecret: string;
     environment: 'client' | 'backend';
-    // Warning: (ae-forgotten-export) The symbol "SimpleFetch" needs to be exported by the entry point index.d.ts
     fetchImplementation?: SimpleFetch;
-    // Warning: (ae-forgotten-export) The symbol "Logger" needs to be exported by the entry point index.d.ts
-    logger?: Logger;
-    region?: 'eu' | 'us';
-    resolveUrl?: string;
     timeout: number;
-}
+    logger?: Logger;
+} & UrlOptions;
 
```

Comment:
Let's actually revert this UrlOptions thing and just have it as two optional properties. I looked a bit deeper into the [TS theory about this](https://stackoverflow.com/questions/76232866/mutually-exclusive-optional-properties), and mutually exclusive properties isn't possible to express cleanly. But more importantly, the user might actually need to supply both! Region is for all endpoints resolve/apply/publish, whereas the new one is only for resolve.